### PR TITLE
Feature/accommodation crud

### PR DESCRIPTION
**Pull request by Sinumi**
Make config file names consistent and added crud operation for creating accommodation

### CODE REVIEW

File: src/api/queries/accommodations/useGetAccommodations.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import httpClient from '@/api/httpClient';
+import { ENDPOINTS } from '@/config/endpoints.config';
+import { AccommodationRes } from '@/types/accommodation.types';
+
+export const useGetAccommodations = () => {
+  return useQuery({
+    queryKey: ['accommodations'],
+    queryFn: () => httpClient.get<AccommodationRes>(ENDPOINTS.accommodation.getAccommodations),
+    // enabled: !!token,
```
Comment (gitkoshelew) [inline src/api/queries/accommodations/useGetAccommodations.ts]: add TODO if you gonna uncomment it some time or remove at all

File: src/pages/accommodations/AccommodationForm.tsx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    },
+    values: accommodation?.data,
+  });
+
+  const onSubmit = (data: AccommodationReq) => {
+    if (id) {
+      if (!Object.keys(dirtyFields).length) {
+        navigate(ROUTES.accommodations.root);
+        return;
+      } else {
```
Comment (gitkoshelew) [inline src/pages/accommodations/AccommodationForm.tsx]: you don't need 'else' if you user return in 'if' statement. remove 'else' from this function at all

**Review by gitkoshelew — CHANGES_REQUESTED** [review]

File: src/pages/accommodations/AccommodationForm.tsx (hunk 2 above)
Comment (Sinumi) [reply src/pages/accommodations/AccommodationForm.tsx]: but if we remove else then it's gonna first updateAccommodation then also tries to createAccommodation

    if (id) { <- Checking if need to update or create
      if (!Object.keys(dirtyFields).length) {  <- Checking if values ever changed so we can avoid unnecessary request 
        navigate(ROUTES.accommodations.root);
        return;
      } else {
        updateAccommodation({
          ...data,
          id,
          availableFrom: dayjs(data.availableFrom).toISOString(),
          availableTo: dayjs(data.availableTo).toISOString(),
        });
      }
    } else {
      createAccommodation({
        ...data,
        availableFrom: dayjs(data.availableFrom).toISOString(),
        availableTo: dayjs(data.availableTo).toISOString(),
      });
    }

File: src/api/queries/accommodations/useGetAccommodations.ts (hunk 1 above)
Comment (Sinumi) [reply src/api/queries/accommodations/useGetAccommodations.ts]: removed

**Pull request closed without merging**